assets/src/edit-story/components/library/panes/text/textSets/loadTextSets.js

```diff
@@ -69,7 +69,7 @@ async function loadTextSet(name) {
 }
 
 export default async function loadTextSets() {
-  const textSets = ['editorial', 'cover'];
+  const textSets = ['editorial', 'cover', 'list_1'];
```

Comment:
Could we rename this to just `list`? All the lists would be under the `list` category, there won't be a separate list 1 and list 2 category, as far as I know.